core/controllers/admin.py

```diff
@@ -70,6 +71,22 @@ def get(self):
         self.render_template('admin-page.mainpage.html')
 
 
+class AdminSessionHandler(webapp2.RequestHandler):
+    """Handler for creating and registering the user"""
+
+    def get(self):
+        auth_claims = auth_services.get_auth_claims_from_request(self.request)
+        if auth_claims is None:
+            auth_services.establish_auth_session(self.request, self.response)
+            self.redirect(self.request.uri)
+        else:
```

Comment:
Let's be safer here and confirm that the user is the super admin:
```suggestion
        elif auth_claims.email != feconf.ADMIN_EMAIL_ADDRESS:
            raise Exception('This endpoint is only accessible to super admin')
```